huggle/Resources/huggle_tips.txt

```diff
@@ -0,0 +1,4 @@
+Huggle contains "EditBar" a little widget that contains both User and Page history in a neat way. It's not enabled by default though. If you are on a small screen, you can replace user and page history widgets with it!
```

Comment:
Huggle contains an "edit bar" - a little widget that contains both User and Page history in a neat way.